eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/soap/SOAPBindingReferenceComposite.java

```diff
@@ -423,21 +429,36 @@ private void bindControls(final DataBindingContext context) {
         ControlDecorationSupport.create(SWTValueUpdater.attach(binding), SWT.TOP | SWT.LEFT);
 
         final IObservableValue msgComposerUnwrappedValue = new WritableValue(realm, null, Boolean.class);
+        final IObservableValue msgComposerCopyNamespacesValue = new WritableValue(realm, null, Boolean.class);
 
         binding = context
                 .bindValue(
                         SWTObservables.observeSelection(_unwrappedPayloadCheckbox), msgComposerUnwrappedValue,
                         new EMFUpdateValueStrategyNullForEmptyString(null, UpdateValueStrategy.POLICY_CONVERT), null);
         ControlDecorationSupport.create(SWTValueUpdater.attach(binding), SWT.TOP | SWT.LEFT);
 
+        org.eclipse.core.databinding.Binding cnBinding = context
+                .bindValue(
+                        SWTObservables.observeSelection(_copyNamespacesCheckbox), msgComposerCopyNamespacesValue,
```

Comment:
You should consider using WidgetProperties for databinding which is the new API (for few years now)
(not blocking for the PR, just to think about it next time yuou manipulate Databinding)